other/tor.md

```diff
@@ -5,8 +5,8 @@ Yggdrasil configuration file to peer with these nodes.
 
 Note that the following assumes Tor is running locally and listening on the default `localhost:9050`.
 
-* HS2 (TCP-only, Hidden Service, v2), operated by [Jeff](https://i2p.rocks/contact.txt)
-  * `socks://localhost:9050/syr3orvfo5nxmao3.onion:40329`
+* HS3 (TCP-only, Hidden Service, v3), operated by [Jeff](https://i2p.rocks/contact.txt)
+  * `socks://localhost:9050/ruoqfb5l2n55x62gu4qsooqm6yamnkjnbqmr6xbxppkxbccgyvioc3ad.onion:1111`
```

Comment:
Could you change the port into something longlived?

```
       LongLivedPorts PORTS
           A list of ports for services that tend to have long-running connections (e.g. chat and interactive shells). Circuits for streams that use these
           ports will contain only high-uptime nodes, to reduce the chance that a node will go down before the stream is finished. Note that the list is
           also honored for circuits (both client and service side) involving hidden services whose virtual port is in this list. (Default: 21, 22, 706,
           1863, 5050, 5190, 5222, 5223, 6523, 6667, 6697, 8300)
```

Reply:
sure thing